cpu/asm/asm.go

```diff
@@ -129,6 +159,11 @@ func signedImm[T int8 | int16, R Imm8 | Imm16](x T) R {
 	return R(x)
 }
 
+func AddLabel(s string, i *Instruction) *Instruction {
```

Comment:
Rename this to `NewLabel`
```suggestion
func NewLabel(s string, i *Instruction) *Instruction {
```